examples/niscope/fetch.py

```diff
@@ -56,83 +57,65 @@
-            niscope_types.GetErrorMessageRequest(vi=vi, error_code=status)
-        )
-        raise Exception(error_message_response.error_message)
-
-
-def check_for_initialization_error(response):
-    """Raise an exception if an error was returned from Initialize."""
-    if response.status < 0:
-        raise RuntimeError(f"Error: {response.error_message or response.status}")
+def check_for_warning(response, vi):
+    """Print to console if the status indicates a warning."""
     if response.status > 0:
-        sys.stderr.write(f"Warning: {response.error_message or response.status}\n")
+        warning_message = niscope_client.ErrorMessage(
+            niscope_types.ErrorMessageRequest(vi=vi, error_code=response.status)
+        )
+        sys.stderr.write(f"{warning_message}\nWarning status: {response.status}\n")
 
 
 try:
     # Open session to NI-SCOPE module with options.
-    init_with_options_response = client.InitWithOptions(
+    init_with_options_response = niscope_client.InitWithOptions(
         niscope_types.InitWithOptionsRequest(
             resource_name=RESOURCE, id_query=False, option_string=OPTIONS
         )
     )
     vi = init_with_options_response.vi
-    check_for_initialization_error(init_with_options_response)
 
     # Configure vertical.
     voltage = 10.0
-    check_for_error(
-        vi,
-        (
-            client.ConfigureVertical(
-                niscope_types.ConfigureVerticalRequest(
-                    vi=vi,
-                    channel_list=CHANNELS,
-                    range=voltage,
-                    offset=0.0,
-                    coupling=niscope_types.VerticalCoupling.VERTICAL_COUPLING_NISCOPE_VAL_AC,
-                    probe_attenuation=1.0,
-                    enabled=True,
-                )
-            )
-        ).status,
+    niscope_client.ConfigureVertical(
+        niscope_types.ConfigureVerticalRequest(
+            vi=vi,
+            channel_list=CHANNELS,
+            range=voltage,
+            offset=0.0,
+            coupling=niscope_types.VerticalCoupling.VERTICAL_COUPLING_NISCOPE_VAL_AC,
+            probe_attenuation=1.0,
+            enabled=True,
+        )
     )
 
     # Configure horizontal timing.
     samples = 1000
-    check_for_error(
-        vi,
-        (
-            client.ConfigureHorizontalTiming(
-                niscope_types.ConfigureHorizontalTimingRequest(
-                    vi=vi,
-                    min_sample_rate=50000000,
-                    min_num_pts=samples,
-                    ref_position=50.0,
-                    num_records=1,
-                    enforce_realtime=True,
-                )
-            )
-        ).status,
+    niscope_client.ConfigureHorizontalTiming(
+        niscope_types.ConfigureHorizontalTimingRequest(
+            vi=vi,
+            min_sample_rate=50000000,
+            min_num_pts=samples,
+            ref_position=50.0,
+            num_records=1,
+            enforce_realtime=True,
```

Comment:
This is existing stuff, but I'm just now realizing how inconsistent the examples are w.r.t. a trailing comma on the last parameter. @reckenro shouldn't `black` be enforcing this one way or the other?

Reply:
FYI, clang-format also uses trailing comma as a hint to put each element on a separate line.

Reply:
I've updated these so any that are not all on a single line are now the trailing comma style since that was the most used.